syncd/ComparisonLogic.cpp

```diff
@@ -61,9 +61,26 @@ ComparisonLogic::ComparisonLogic(
 
     // TODO needs to be removed and done in generic
 
-    m_current->m_defaultTrapGroupRid     = m_switch->getSwitchDefaultAttrOid(SAI_SWITCH_ATTR_DEFAULT_TRAP_GROUP);
-    m_temp->m_defaultTrapGroupRid        = m_switch->getSwitchDefaultAttrOid(SAI_SWITCH_ATTR_DEFAULT_TRAP_GROUP);
+    try
```

Comment:
Why tis is modified?

Reply:
SAI_SWITCH_ATTR_DEFAULT_TRAP_GROUP is not supported by Credo gbsyncd.
This is to avoid raise exception for SAI_SWITCH_ATTR_DEFAULT_TRAP_GROUP not supported